mitodl/ocw-studio
# Update various pages to use new, Card-based designs

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #747

#### What's this PR do?

This PR implements the new Card design, where instead of having a card with a list of things inside of it, we want to have a list of cards. Check the issue for details and links to designs and so on.

#### How should this be manually tested?

Check out the various pages that are relevant: 

- the site index  page (`/sites/`)
- site content page (`/sites/foobar/type/contenttype/`)
- collaborators page (`/sites/foobar/collaborators/`)

On all of those you should see a list of cards, and the changes specified in the issue should be made as well (like adding the dropdown menu on the collaborators page).

#### Screenshots (if appropriate)

<img width="1677" alt="Screen Shot 2021-11-04 at 4 41 29 PM" src="https://user-images.githubusercontent.com/6207644/140417025-9972e07d-9b5c-487c-aeb3-5e8164c2cec6.png">
<img width="1676" alt="Screen Shot 2021-11-04 at 4 42 53 PM" src="https://user-images.githubusercontent.com/6207644/140417030-ad11ac89-03c0-4749-8bd6-06dfc7bf8a49.png">
<img width="1680" alt="Screen Shot 2021-11-04 at 4 41 43 PM" src="https://user-images.githubusercontent.com/6207644/140417031-62c0f874-e26b-4be0-aa1d-cb5b70e850a2.png">
<img width="1680" alt="Screen Shot 2021-11-04 at 4 41 34 PM" src="https://user-images.githubusercontent.com/6207644/140417033-3bc898a9-5fe6-4b2a-a771-c1813e63b732.png">
